### PR TITLE
Add YAML schedule for TW zdup-Leap-42.3-gnome

### PR DESCRIPTION
**Pull request by Zaoliang**
remove opensuse_welcome from zdup-Leap-42.3-gnome
see https://progress.opensuse.org/issues/56774

### CODE REVIEW

**Comment by Zaoliang** [comment]
verification test:

http://f40.suse.de/tests/7425

**Comment by SergioAtSUSE** [comment]
## All this jobs are scheduling the module **opensuse_welcome**

#### O3
- TW x86_64: https://openqa.opensuse.org/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=microos&distri=opensuse&version=Tumbleweed&build=20200413&groupid=1#
- Leap 15.2 x86_64: https://openqa.opensuse.org/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=opensuse&version=15.2&build=626.2&groupid=50#

#### OSD
- Functional: none found https://openqa.suse.de/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=sle&version=15-SP2&build=176.1&groupid=110#
- Desktop: none found https://openqa.suse.de/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=sle&version=15-SP2&build=176.1&groupid=118#
- Migration: none found https://openqa.suse.de/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=sle&version=15-SP2&build=176.1&groupid=265#
- Yast: none found https://openqa.suse.de/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=sle&version=15-SP2&build=176.1&groupid=129#

File: lib/main_common.pm [hunk 1]
```diff
@@ -481,7 +481,6 @@ sub load_zdup_tests {
     # Restrict version switch to sle until opensuse adopts it
     loadtest "migration/version_switch_upgrade_target" if is_sle and get_var("UPGRADE_TARGET_VERSION");
     loadtest 'boot/boot_to_desktop';
-    loadtest "installation/opensuse_welcome"  if opensuse_welcome_applicable();
```
Comment (SergioAtSUSE) [inline lib/main_common.pm:484]: This cannot be done until **first_boot** handles the popup: https://openqa.opensuse.org/tests/1234107

Or all those tests will fail

**Comment by SergioAtSUSE** [comment]
The way to go should be to create YAML schedules for those test suites with the current schedule (including opensuse_welcome).
After that, we can safely unschedule the module **opensuse_welcome** only on the test suites where we don't need it.

File: lib/main_common.pm (hunk 1 above)
Comment (Zaoliang) [reply lib/main_common.pm:484]: > This cannot be done until **first_boot** handles the popup: https://openqa.opensuse.org/tests/1234107#step/opensuse_welcome/5
> 
> Or all those tests will fail

actually this is working for opensuse_welcome, confusing...

**Comment by Zaoliang** [comment]
> ## All this jobs are scheduling the module **opensuse_welcome**
> #### O3
> * TW x86_64: https://openqa.opensuse.org/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=microos&distri=opensuse&version=Tumbleweed&build=20200413&groupid=1#
> * Leap 15.2 x86_64: https://openqa.opensuse.org/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=opensuse&version=15.2&build=626.2&groupid=50#
> 

All passed for opensuse_welcome, I don't understand what it is needed to change.

> #### OSD
> * Functional: **none found** https://openqa.suse.de/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=sle&version=15-SP2&build=176.1&groupid=110#
> * Desktop: **none found** https://openqa.suse.de/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=sle&version=15-SP2&build=176.1&groupid=118#
> * Migration: **none found** https://openqa.suse.de/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=sle&version=15-SP2&build=176.1&groupid=265#
> * Yast: **none found** https://openqa.suse.de/tests/overview?arch=&machine=&modules=opensuse_welcome&distri=sle&version=15-SP2&build=176.1&groupid=129#

opensuse_welcome is openSUSE related, not needed for SLES, Why?? 
really confusing...

**Comment by SergioAtSUSE** [comment]
> opensuse_welcome is openSUSE related, not needed for SLES, Why??
> really confusing...

Yes, sorry, my mistake, I don't know what I was thinking about...
We can ignore Leap tests, since they don't have zdup tests.

But, there are still a few zdup tests in Tumbleweed:
- https://openqa.opensuse.org/tests/1234718#step/opensuse_welcome/5
- https://openqa.opensuse.org/tests/1234719#step/opensuse_welcome/5
- https://openqa.opensuse.org/tests/1234593#step/opensuse_welcome/5
- https://openqa.opensuse.org/tests/1234594#step/opensuse_welcome/5
- https://openqa.opensuse.org/tests/1234725#step/opensuse_welcome/5
- https://openqa.opensuse.org/tests/1234894#step/opensuse_welcome/5
- https://openqa.opensuse.org/tests/1234582#step/opensuse_welcome/5

So, the module to adapt should be boot_to_desktop, but, since the opensuse_welcome is scheduled just after it, it is ok. So, the proper way for zdup tests is to create a YAML schedule where opensuse_welcome is explicitely scheduled.

Your [verification run](http://f40.suse.de/tests/7425#step/boot_to_desktop/4) hit the Wayland/GNOME bug: https://bugzilla.opensuse.org/show_bug.cgi?id=1169203
The popup didn't appear, but, it should and it does in production: https://openqa.opensuse.org/tests/1234725#step/boot_to_desktop/4

**Comment by Zaoliang** [comment]
verification test:

http://f40.suse.de/tests/7442

**Comment by SergioAtSUSE** [comment]
I noticed a typo in the commit description: `zdup--Leap-42.3-gnome` (two dashes)
After that I think we are safe to merge. :)

**Comment by Zaoliang** [comment]
typo corrected now, thanks

**Pull request merged**